tests/test_vector2_normalize.py

```diff
@@ -0,0 +1,14 @@
+import ppb_vector
+import pytest
+
+
+@pytest.mark.parametrize("x, y, expected", [
+    (3, 4, 1),
```

Comment:
This is interesting and I like the tests, but I'd like to include your tests with #34 (as they test separate things). Can you change the name of this test to `test_normalize_length`.